docs/source/user/aerodyn/theory.rst

```diff
@@ -37,5 +37,73 @@ Eames tower shadow model (**TwrShadow=2**) is given by:
 where :math:`TI` is the turbulence intensity at the tower node. 
 
 
-
-
+.. _AD_buoyancy:
+
+Buoyancy
+~~~~~~~~
+
+When a solid object is submerged in a fluid, it experiences a net force, buoyancy, 
+from the hydrostatic fluid pressure acting on its surface. This force can often 
+be neglected in less dense fluids, such as air, but can be significant in denser
+fluids, such as water. To capture the effects of this force on MHK turbines, 
+buoyant loads are calculated for the turbine blades, tower, hub, and nacelle. 
+Marine growth is neglected for all components. :numref:`AD_buoy_coords` - 
+:numref:`AD_buoy_hubnacelle` detail the coordinate systems and blade, tower, hub,
+and nacelle buoyancy calculations.
+
+.. _AD_buoy_coords:
+
+Coordinate Systems
+------------------
+The buoyant force acting on an element depends on its instantaneous orientation 
+and depth. The orientation is defined by heading and inclination angles, which
+are calculated for each element at every time step. Total water depth is defined
+by the user, relative to the mean sea level. The instantaneous depth of each
+element is based on the total water depth and element position and is calculated
+at each time step.
```

Comment:
While the AeroDyn driver doesn't need MSL2SWL, the OpenFAST driver and HydroDyn support the latter.  When MSL2SWL = 0, the above statements are correct.  When MSL2SWL /= 0, then the above statements should refer to the still water level rather than the mean sea level.  You may want to update the wording above a little to clarify.